apps/dotcom/client/src/tla/components/TlaEditor/TlaEditor.tsx

```diff
@@ -207,8 +210,19 @@ export function TlaEditor({
 		}
 	}, [app, fileId])
 
+	const user = useTldrawUser()
+
 	const store = useSync({
-		uri: `${MULTIPLAYER_SERVER}/app/file/${fileSlug}`,
+		uri: useCallback(async () => {
+			const url = new URL(`${MULTIPLAYER_SERVER}/app/file/${fileSlug}`)
+			if (user) {
+				url.searchParams.set('accessToken', await user.getToken())
+			}
+			if (temporary) {
```

Comment:
I feel like when I see it say 'temporary' it makes me think the room will disappear after a certain time period. Would 'sandbox' / 'playground' be better options in general?

Reply:
you can change the name later if you like. make sure you bump the documentInfo version number in the durable object. i'm fine with any of these.